_ont/ont-huawei-ma5671a-root-web.md

```diff
@@ -0,0 +1,317 @@
+
+| USB TTL (UART) Adapter | wire colour in picture | SFP 20pins Molex connector |
+| ---------------------- | ---------------------- | -------------------------- |
+| 3.3V                   | red                    | pin #15 and #16            |
+| TX                     | orange                 | pin #2                     |
+| RX                     | yellow                 | pin #7                     |
+| GND                    | green                  | pin #14                    |
+
+{% include image.html file="ma5671a-root-1.jpg"  alt="Example of how the SFP-TTL connection should look like" caption="Example of how the sfp-ttl connection should look like" %}
+{% include image.html file="new-root-procedure\board-molex-arduino.jpg"  alt="Example of how the SFP-TTL connection should look like with a custom board" caption="Example of how the SFP-TTL connection should look like with a custom board" %}
+
+{% include alert.html content="Try PIN 10 or other GND PINs if the connection doesn't work with PIN 14." alert="Note" icon="svg-warning" color="yellow" %}
+
+{% include alert.html content="Some USB TTL adapters label TX and RX pins the other way around: try to swap them if the connection doesn't work." alert="Note"  icon="svg-warning" color="yellow" %}
+
+Connect the TTL adapter to the computer, once done press the following button <button id="start-button" class="btn" data-toogle="modal" data-target="#root-modal" disabled>Start root!</button>. A window will open that will execute the root.
+
+<script>
+if ('serial' in navigator) {
+    document.getElementById('start-button').disabled = false;
+} else {
+    document.getElementById('browser-error').style.display = 'block';
+}
+</script>
+<p id="browser-error" style="display:none">{% include alert.html content="This browser is not compatible with the web-root procedure. See the <a href='https://developer.mozilla.org/en-US/docs/Web/API/Web_Serial_API#browser_compatibility'>Browser compatibility</a>" alert="Note"  icon="svg-warning" color="red" %}</p>
+<noscript>
+{% include alert.html content="Your browser does not support JavaScript!" alert="Note"  icon="svg-warning" color="red" %}
+</noscript>
+
+
+# Connect to the stick via SSH
+
+After this is done, reboot the stick, after connecting it to the router via an ethernet mediaconverter or directly plug it in an SFP port, with the port's IP set to whatever IP of the `192.168.1.0/24` subnet (the stick has the IP `192.168.1.10`)
+
+{% include alert.html content="If your subnet is `192.168.1.0/24` make sure you have no ip conflicts." alert="Note" icon="svg-warning" color="yellow" %}
+
+{% include alert.html content="Make sure to disable SFP TX fault detection, otherwise the RX loss will prevent you from connecting to the mini SFP ONT at this point. Don't simply attach the fiber cable to work around this issue as the OLT may ban you." alert="Note" icon="svg-warning" color="yellow" %}
```

Comment:
Perhaps it would be good to indicate that in media converters and on mikrotik it is not necessary to disable tx fault

Reply:
Sure!